webapp/src/pages/Users/Users.tsx

```diff
@@ -66,11 +66,7 @@ export default function Users() {
         </Link>
       </div>
       <div className="users-search">
-        <Input
-          iconVariant={IconVariant.SEARCH}
-          variant={InputVariant.DARK}
-          placeholder="search"
-        />
+        <SearchUserForm search={search} setSearch={setSearch} />
```

Comment:
thought: The way it works now, `SearchUserForm` is a component that sets a custom-styled Input.

suggestion (non-blocking): We could extract some functionality from the `Users` page and encapsulate it into this component.
To do this, we could move the `search` and `setSearch` inside the `SearchUserForm` and pass the `setUsersList` as a prop.
This is what I mean https://stackoverflow.com/a/71836971
And if we want to make it a generic component, we could also pass the URL(or maybe the fetching function?) as a prop.

Reply:
I've update it to searchForm in a more generic way, but please have a look and see which way you like it the most

Reply:
Thank you! I like it more now because we could reuse this component to search games, chat channels, etc.